src/pages/locations/index.tsx

```diff
@@ -84,11 +132,67 @@ export const LocationsList: React.FC<IDataModelListProps> = (props) => {
           hideCreateButton={props.hideCreateButton}
         />
       ) : (
-        <GoogleMapContainer
-          markers={markers}
-          defaultCenter={{ lat: 36.7783, lng: -119.4179 }}
-          zoom={6}
-        />
+        <div
+          style={{
+            width: '100%',
+            textAlign: 'center',
+            alignItems: 'center',
+            height: defaultHeight,
+            justifyContent: 'center',
+            marginBottom: '60px',
+          }}
+        >
+          <div
+            style={{
+              position: 'relative',
+            }}
+          >
+            <input
```

Comment:
what do you think about using <AutoComplete /> here? - https://ant.design/components/auto-complete